unit-tests/test-fw-update.py

```diff
@@ -74,7 +74,13 @@ def reset_update_counter( device ):
 
     send_hardware_monitor_command( device, cmd )
 
-def find_image_or_exit( product_name, image_mask ):
+def find_image_or_exit( product_name, fw_version ):
+    """
+    Description...
+    :param product_name: the name of the camera
+    :param fw_version: fw version
```

Comment:
fw_version?
Isn't this a regular expression?

Reply:
can be in the case of recovery mode when we dont have recommended_firmware_version

Reply:
So it should say so?